src/symmeplot/matplotlib/scene.py

```diff
@@ -144,6 +147,20 @@ def plot(self, prettify=True, ax_scale=1.5):
             self.auto_zoom()
             self.axes.set_aspect("equal", adjustable="box")
 
+    def as_orthogonal_projection_plot(self, frame: ReferenceFrame | None = None) -> None:
+        """Change the axis to an orthogonal projection making the view seemingly 2D.
+
+        Parameters
+        ----------
+        frame : ReferenceFrame, optional
+            Reference frame w.r.t. which the axis view is oriented aligning the users
+            view with the ZY plane. The default is the inertial frame of the scene.
```

Comment:
```suggestion
            view with its YZ plane. The default is the inertial frame of the scene.
```
"_its_" might make it a bit clearer.